server/etcdserver/api/rafthttp/transport.go

```diff
@@ -339,24 +339,31 @@ func (t *Transport) RemoveAllPeers() {
 
 // the caller of this function must have the peers mutex.
 func (t *Transport) removePeer(id types.ID) {
-	if peer, ok := t.peers[id]; ok {
+	// etcd may remove a member again on startup due to WAL files replaying.
+	// (see: https://github.com/etcd-io/etcd/pull/13479)
```

Comment:
The PR number (13479) changes. Suggest to remove the  `// (see: https://github.com/etcd-io/etcd/pull/13479)`.